webapp/app/js/perf.js

```diff
@@ -0,0 +1,388 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, you can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use strict";
+
+var perf = angular.module("perf", ['ui.router', 'ui.bootstrap']);
+
+/* Copied from providers.js */
+perf.provider('thServiceDomain', function() {
+    this.$get = function() {
+        if (window.thServiceDomain) {
+            return window.thServiceDomain;
+        } else {
+            return "";
+        }
+    };
+});
+
+var SERIES_COLORS = [ 'red', 'green', 'blue', 'orange', 'purple' ];
```

Comment:
can you wrap it in a value definition? See https://github.com/mozilla/treeherder-ui/blob/master/webapp/app/js/values.js for an example